vda5050_safety_state_broadcaster/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vda5050_safety_state_broadcaster</name>
+  <version>0.1.0</version>
```

Comment:
```suggestion
  <version>6.1.0</version>
```
needs to have the same version as other packages in the repository, or the release process will fail.